test/integration/targets/command_shell/tasks/main.yml

```diff
@@ -282,6 +320,22 @@
     path: "{{ output_dir_test }}/afile.txt"
     state: file
 
+# changes
+
+- name: Verify that afile.txt content has changed
+  assert:
+    that: "'win' in lookup('file', '{{ output_dir_test }}/afile.txt') | b64encode | b64decode"
+
+- name: Execute the test.sh script with chdir
+  shell: "{{ output_dir_test }}/change_test.sh > {{ output_dir_test }}/afile.txt"
+  args:
+    chdir: "{{ output_dir_test }}"
+    changes: "{{ output_dir_test }}/afile.txt"
+
+- name: Verify that afile.txt content has changed
+  assert:
+    that: "'won' in lookup('file', '{{ output_dir_test }}/afile.txt') | b64encode | b64decode"
+
```

Comment:
How come these are at the bottom? Why not group them with the other changes tests?

Reply:
This `main.yml` file is getting a bit long and hard to follow. It's probably a good idea to put these new tests in a separate task file and use `import_tasks` to run them. The rest of the test cases should be broken into separate tasks files, too, but not in this PR.

Reply:
@samdoran I'm realizing that we have this amount of test in some of the modules already in place. Could be a good idea to work on it and split them in different files per functionality using different PRs? Of course, I will do for this one, but I had the same comment with you for `lineinfile` -> https://github.com/ansible/ansible/pull/70647

What do you think?